data/calls_and_jobs.json

```diff
@@ -1,6 +1,14 @@
 {
     "last_updated": "2022-11-07",
```

Comment:
Maybe we don't need this at the moment, can you remove it for me. (Sorry had that thinking we might use it, but we can just put it whenever we decide to use it)

Reply:
Done!